### DISCUSSION TITLE
How to generate a tessellation with bimodal grains?

### DISCUSSION THREAD

**Discussion by xiamucp** [neperfepx/neper#671] [Q&A]
I hope all is well. I wonder how can I generate a tessellation like this?
![image](https://user-images.githubusercontent.com/102226925/227780696-e953c2dd-0cb2-4d83-8928-a08626c12462.png)

**Comment by rquey** [comment]
Provide a (weighted) sum of distributions in `-morpho`:
```
neper -T -n 100 -morpho "diameq:lognormal(0.1,0.01)+0.1*lognormal(0.2,0.02),1-sphericity:lognormal(0.15,0.03)" -dim 2 -morphooptiini "weight:radeq,coo:packing"
```

The `-morphooptiini "weight:radeq,coo:packing"` option speeds  up the process using a better initial solution.

![image](https://user-images.githubusercontent.com/2659376/227858022-1ec3d640-70f3-4fdb-83e2-2c11917d70c9.png)
